types/commander/index.d.ts

```diff
@@ -17,7 +16,7 @@ declare namespace commander {
         unknown: string[];
     }
 
-    class Option {
+    interface Option {
```

Comment:
this does not look right. `Option` seems to be a class, and not an object that is callable with `new`:

```sh
> var commander = require("commander");
undefined
> commander.Option
[Function: Option]
> new commander.Option("--f");
Option {
  flags: '--f',
  required: 0,
  optional: 0,
  bool: true,
  long: '--f',
  description: '' }
> new (new commander.Option("--f"))("--f");
TypeError: (intermediate value) is not a constructor
    at repl:1:1
    at realRunInThisContextScript (vm.js:22:35)
    at sigintHandlersWrap (vm.js:98:12)
    at ContextifyScript.Script.runInThisContext (vm.js:24:12)
    at REPLServer.defaultEval (repl.js:346:29)
    at bound (domain.js:280:14)
    at REPLServer.runBound [as eval] (domain.js:293:12)
    at REPLServer.onLine (repl.js:544:10)
    at emitOne (events.js:101:20)
    at REPLServer.emit (events.js:188:7)
```